css/styles.css

```diff
@@ -138,3 +138,97 @@ nav a {
   cursor: pointer;
   font-size: 20px;
 }
+
+/* Modal */
+
+.modal-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+}
+
+.modal-content {
+  background-color: #054d28;
+  text-align: center;
+  width: 400px;
+  height: 200px;
+  border-radius: 5px;
+  color: white;
+}
+
+.modal-content h3 {
+  font-size: 30px;
+  font-family: sans-serif;
+  margin-bottom: 50px;
+}
```

Comment:
For small sizes (fonts, margins, etc.), please use `rem` over `px`: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css